### PR TITLE
Add Pagination Error Statement / Example

### PR DESCRIPTION
**Pull request by travisgosselin**
Clarity around the expectation on pagination input validation failure is added and linked to the errrors page.

### CODE REVIEW

File: standards/collections.md [hunk 1]
```diff
@@ -69,6 +70,7 @@ POST https://api.spscommerce.com/v1/books
      &offset=int                               
      &cursor=string
 // REQUEST
+HTTP/1.1 200
```
Comment (greylurk) [inline standards/collections.md:73]: Shouldn't this be 
```
POST https://api.spscommerce.com/v1/books
{
  "paging": {
  ...
}

Comment (travisgosselin) [reply standards/collections.md:73]: ah looks like I copy pasted into the request thinking it was a response. Thanks!

**Review by ravisaluru — APPROVED** [review]

File: standards/collections.md [hunk 2]
```diff
@@ -112,6 +116,34 @@ GET https://api.spscommerce.com/v1/example?limit=25&offset=0
...
+    "detail": "Missing content or invalid input provided.",
+    "instance": "/v1/books",
+    "requestId": "b6d9a290-9f20-465b-bcd3-4a5166eeb3d7",
+    "context": [
+       {
+            "code": "INPUT_MIN_VALUE",
+            "message": "Attribute 'limit' must be greater than or equal to 1.",
+            "field": "limit",
+            "source": "query",
+            "value": "0"
```
Comment (mkokotovich) [inline standards/collections.md:140]: Wouldn't this value be `-2` for this example? Or maybe I don't remember what the `value` field is used for :)

File: standards/collections.md [hunk 3]
```diff
@@ -112,6 +116,34 @@ GET https://api.spscommerce.com/v1/example?limit=25&offset=0
 }
 ```
 
+- Invalid requests due to pagination validation **MUST** return a `400 Bad Request` status code following the standard [error schema](errors.md).
```
[the remark this answers is not in the document]
Comment (travisgosselin) [reply standards/collections.md:119]: Good call.

File: standards/collections.md (hunk 2 above)
Comment (travisgosselin) [reply standards/collections.md:140]: haha, yep totally.

**Review by greylurk — APPROVED** [review]
Looks good to me.

**Pull request merged**